db/migrate/20200519141015_create_storage_resources.rb

```diff
@@ -0,0 +1,15 @@
+class CreateStorageResources < ActiveRecord::Migration[5.1]
+  def change
+    create_table :storage_resources do |t|
+      t.string :name
+      t.references :ems, :type => :bigint, :index => true, :references => :ext_management_system
+      t.string :ems_ref
+      t.string :uuid
+      t.bigint :logical_free
+      t.bigint :logical_total
+      t.string :pool_name
```

Comment:
If this is a storage pool, and it has a name, what is the `pool_name` then?

Reply:
This was added for convenience as the resource name can possibly be different then the pool name. (but a resource may not necessarily be a pool.) we'll remove the pool_name from the storage_resource.